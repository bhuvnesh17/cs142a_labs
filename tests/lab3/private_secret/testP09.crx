// test many methods, each with problem


func main(a:void) : int {
  var a : int;
  ::println(b);
}
func main2(a:void) : int {
  var a : int;
  ::println(b);
}
func main3(a:void) : int {
  var a : int;
  ::println(b);
}
func main4(a:void) : int {
  var a : int;
  ::println(b);
}
func main5(a:void) : int {
  var a : int;
  ::println(b);
}
func main6(a:void) : int {
  var a : int;
  ::println(b);
}
func main7(a:void) : int {
  var a : int;
  ::println(b);
}
func main8(a:void) : int {
  var a : int;
  ::println(b);
}
func main9(a:void) : int {
  var a : int;
  ::println(b);
}
