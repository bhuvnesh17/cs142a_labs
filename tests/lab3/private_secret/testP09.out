Error parsing file.
DeclareSymbolError(5,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a)

ResolveSymbolError(6,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a)

DeclareSymbolError(9,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
  Symbol(a)

ResolveSymbolError(10,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
  Symbol(a)

DeclareSymbolError(13,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
  Symbol(a)

ResolveSymbolError(14,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
  Symbol(a)

DeclareSymbolError(17,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
  Symbol(a)

ResolveSymbolError(18,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
  Symbol(a)

DeclareSymbolError(21,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
  Symbol(a)

ResolveSymbolError(22,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
  Symbol(a)

DeclareSymbolError(25,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
Symbol(main6)
  Symbol(a)

ResolveSymbolError(26,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
Symbol(main6)
  Symbol(a)

DeclareSymbolError(29,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
Symbol(main6)
Symbol(main7)
  Symbol(a)

ResolveSymbolError(30,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
Symbol(main6)
Symbol(main7)
  Symbol(a)

DeclareSymbolError(33,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
Symbol(main6)
Symbol(main7)
Symbol(main8)
  Symbol(a)

ResolveSymbolError(34,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
Symbol(main6)
Symbol(main7)
Symbol(main8)
  Symbol(a)

DeclareSymbolError(37,7)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
Symbol(main6)
Symbol(main7)
Symbol(main8)
Symbol(main9)
  Symbol(a)

ResolveSymbolError(38,13)[Could not find b.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
Symbol(main2)
Symbol(main3)
Symbol(main4)
Symbol(main5)
Symbol(main6)
Symbol(main7)
Symbol(main8)
Symbol(main9)
  Symbol(a)


